tests/baselines/reference/extendAndImplementTheSameBaseType2.errors.txt

```diff
@@ -16,9 +15,8 @@ tests/cases/compiler/extendAndImplementTheSameBaseType2.ts(16,5): error TS2322:
     class D extends C<string> implements C<number> {
           ~
 !!! error TS2720: Class 'D' incorrectly implements class 'C<number>'. Did you mean to extend 'C<number>' and inherit its members as a subclass?
-!!! error TS2720:   Types of property 'bar' are incompatible.
-!!! error TS2720:     Type '() => string' is not assignable to type '() => number'.
-!!! error TS2720:       Type 'string' is not assignable to type 'number'.
+!!! error TS2720:   The types of 'bar()' are incompatible between these types.
+!!! error TS2720:     Type 'string' is not assignable to type 'number'.

```

Comment:
Is it easy/feasible to be able to add a modifier like "return" in a case like this?

```
!!! error TS2720:   The return types of 'bar()' are incompatible between these types.
	!!! error TS2720:     Type 'string' is not assignable to type 'number'.
```

Reply:
Maybe

```
The types returned by '{0}' are incompatible between these types.
```

when dealing with a call/construct?